src/esw/brushless.py

```diff
@@ -119,12 +120,18 @@ async def _send_command(self) -> None:
                 ),
                 timeout=self.MOTEUS_RESPONSE_TIME_INDICATING_DISCONNECTED_S,
             )
+
+            if state is None or not hasattr(state, "values") or moteus.Register.FAULT not in state.values:
```

Comment:
either add a comment or just set this statement to a variable called moteus_not_found